sphinxcontrib/chapeldomain/chapel.py

```diff
@@ -109,7 +117,18 @@ class ChapelLexer(RegexLexer):
             (r'[a-zA-Z_][\w$]*', Name.Class, '#pop'),
         ],
         'procname': [
-            (r'([a-zA-Z_][.\w$]*|\~[a-zA-Z_][.\w$]*|[+*/!~%<>=&^|\-]{1,2})',
+            (r'([a-zA-Z_][.\w$]*|'    # regular function name, including 2ndary
```

Comment:
Please write "secondary" instead of "2ndary" XD It's be one thing if it rendered with "nd" above and "ary" on the same line, but it throws me off and it really doesn't save much space.

Reply:
I don't remember the mindset that I was in when I typed `2ndary`. I'd concur that it looks disgusting.